scaling_performance/optimizing_storage.adoc

```diff
@@ -13,6 +13,81 @@ toc::[]
 
 == Optimizing Storage
 
+== General Storage Guidance
+
+The following is a matrix which details the storage backend types {product-title} can be used with implementation examples and vendors.
+
+=== Available Storage Options
+[format="csv",width="70%",cols="3"]
+[options="header]
+|=======================
+Type,Examples,Vendors
+hostPath,"hostPath pod mounted volumes.  A specific type of Block Storage.  File paths on the node mounted onto running pods.",Others
```

Comment:
@raffaelespazzoli Thanks for this.  I intended to only speak to Persistent Storage Types.  I need to speak with you (and others) more about ephemeral storage, because, while it may have benefits I assume its supportability is much different that Persistent Storage Types.  You can always add an entry to the spreadsheet I linked to as well.